tests/Fixer/ClassNotation/NullPropertyDeclarationFixerTest.php

```diff
@@ -20,6 +20,8 @@
  * @internal
  *
  * @covers \PhpCsFixer\Fixer\ClassNotation\NullPropertyDeclarationFixer
+ *
+ * @group new
```

Comment:
i bet this is temporary. you could achieve same with `--filter` param without modifying the code itself